stl/inc/xmemory

```diff
@@ -2786,6 +2787,15 @@ _NODISCARD _Elem* _UIntegral_to_buff(_Elem* _RNext, _UTy _UVal) { // used by bot
     auto _UVal_trunc = static_cast<unsigned long>(_UVal);
 #endif // ^^^ !defined(_WIN64) ^^^
 
+    if constexpr (_Is_any_of_v<_Elem, char, wchar_t>) {
+        while (_UVal_trunc >= 100) {
+            const unsigned long _UVal_trunc_part = _UVal_trunc % 100;
+            _UVal_trunc /= 100;
+            *--_RNext = static_cast<_Elem>(__DIGIT_TABLE<_Elem>[_UVal_trunc_part * 2 + 1]);
+            *--_RNext = static_cast<_Elem>(__DIGIT_TABLE<_Elem>[_UVal_trunc_part * 2]);

```

Comment:
This can be further optimized to
```c++
            _RNext -= 2;
            _CSTD memcpy(_RNext, __DIGIT_TABLE<_Elem> + _UVal_trunc_part * 2, 2 * sizeof(_Elem))
```
still without much better results in the benchmark.